django/db/backends/oracle/schema.py

```diff
@@ -21,6 +21,12 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_create_column_inline_fk = 'CONSTRAINT %(name)s REFERENCES %(to_table)s(%(to_column)s)%(deferrable)s'
     sql_delete_table = "DROP TABLE %(table)s CASCADE CONSTRAINTS"
     sql_create_index = "CREATE INDEX %(name)s ON %(table)s (%(columns)s)%(extra)s"
+    sql_create_table_with_comment = """
+        CREATE TABLE %(table)s (%(definition)s);
+        COMMENT ON TABLE %(table)s IS '%(table_comment)s'
+    """
```

Comment:
I'd rather execute additional statement based on `sql_create_table_comment` instead of adding another statement.